code_schemes/kakuma_s01e05_reasons.json

```diff
@@ -3,6 +3,78 @@
   "Name": "KAKUMA S01E05 Reasons",
   "Version": "0.0.0.1",
   "Codes": [
+    {
```

Comment:
The order of codes here differs from the doc

Reply:
sorted edu - codes together for easy labeling. RDA request